comp/dogstatsd/listeners/telemetry_windows.go

```diff
@@ -1,60 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-//go:build windows
-
-package listeners
-
-import (
-	"expvar"
-	"fmt"
-
-	"github.com/DataDog/datadog-agent/pkg/telemetry"
-)
-
-type listenerTelemetry struct {
-	packetReadingErrors expvar.Int
-	packets             expvar.Int
-	bytes               expvar.Int
-	expvars             *expvar.Map
-	tlmPackets          telemetry.Counter
-	tlmPacketsBytes     telemetry.Counter
-}
-
-func newListenerTelemetry(metricName string, name string) *listenerTelemetry {
```

Comment:
I've moved this functionality to `telemetry.go` since the UDP listeners can benefit from having this code available as well.